rules_building_block/defense_evasion_dll_hijack.toml

```diff
@@ -35,7 +35,13 @@ library where host.os.type == "windows" and
     process.code_signature.status :"trusted" and not dll.code_signature.status : ("trusted", "errorExpired", "errorCode_endpoint*") and
     /* DLL loaded from the process.executable current directory */
     endswith~(substring(dll.path, 0, length(dll.path) - (length(dll.name) + 1)), substring(process.executable, 0, length(process.executable) - (length(process.name) + 1)))
-    and not user.id : "S-1-5-18"
+    and not user.id : "S-1-5-18" and
+  not dll.name : (
```

Comment:
I know its noisy, but this will weaken the BBR coverage, maybe excluding top hashes, just saw a TP  this morning in detonate with ffmpeg.dll

Reply:
Updated the PR excluding the dll hashes for the ones which:
* Have 0 detections in VT
* >= 15 clusters in telem
* Good VT community Score